sklearn/covariance/_shrunk_covariance.py

```diff
@@ -686,14 +691,19 @@ def fit(self, X, y=None):
         self._validate_params()
 
         X = self._validate_data(X)
```

Comment:
Oh right. We need to change the line here:

```suggestion
        X = self._validate_data(X, ensure_2d=False)
```

We can see in another PR if we want to deprecate the full behaviour of passing 1D array.